packages/std/Cargo.toml

```diff
@@ -30,6 +30,8 @@ stargate = []
 [dependencies]
 base64 = "0.13.0"
 cosmwasm-derive = { path = "../derive", version = "0.14.0" }
+getrandom = { version = "0.2.3", features = ["js"] }
```

Comment:
Why `getrandom`?

Reply:
It's a dependency of `primitive_types`. I needed to enable the `js` feature for it so that wasm builds still work.

Reply:
Hmm, any chance to get rid of the rand dependency by reducing features?

Also can we use the lower level `uint` create instead?

Reply:
It didn't look like the `uint` crate was published to crates.io. I'll double check.

Reply:
Oh, I was wrong: https://crates.io/crates/uint